lib/modsulator/normalizer.rb

```diff
@@ -165,12 +174,12 @@ def trim_text(node)
 
   # Removes the point attribute from single <dateCreated> and <dateIssued> elements.
   #
-  # @param [Nokogiri::XML::Element]   root  The root of a MODS XML document.
+  # @param [Nokogiri::XML::NodeSet]   nodes  A set of all affected <dateCreated> and <dateIssued> elements.
   # @return [Void]                    The given document is modified in place.
-  def clean_date_attributes(root)
-    
+  def clean_date_attributes(nodes)
+
     # Find all the <dateCreated> and <dateIssued> elements that are NOT immediately followed by another element with the same name
-    root.xpath('//mods:originInfo/mods:dateCreated[1][not(following-sibling::*[1][self::mods:dateCreated])] | //mods:originInfo/mods:dateIssued[1][not(following-sibling::*[1][self::mods:dateIssued])]', 'mods' => 'http://www.loc.gov/mods/v3').each do |current_element|
+    nodes.each do |current_element|
       attributes = current_element.attributes
       if(attributes.key?('point'))
```

Comment:
This bit is still tied to the MODS schema I think? Not sure what the `point` attribute is.